CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Recommendation: for ease of reading, use the following order:
 - Fixed
 -->
 
+## [Unreleased]
+### Changed
+- `kamu push <dataset>` command now can be called without `--to` reference and Alias or Remote dataset repository will be used as destination
+- `kamu login` command now will store repository to Repository registry. Name can be provided with `--repo-name` flag and to skip creating repo can be used `--skip-add-repo` flag
```

Comment:
We also need to check if our instructions on pulling/pushing datasets in the Jupyter demo notebooks need to be updated or can be simplified.

Reply:
Updated